packages/patternfly-3/patternfly-react-extensions/src/components/CatalogTile/CatalogTile.stories.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info/dist/index';
+import { defaultTemplate } from 'storybook/decorators/storyTemplates';
+import { storybookPackageName } from 'storybook/constants/siteConstants';
+import pfBrand from 'patternfly/dist/img/logo.svg';
+import github from 'patternfly/dist/img/github-logo.svg';
+import { Icon } from 'patternfly-react';
+import { CatalogTile, CatalogTileBadge } from './index';
+
+import { name } from '../../../package.json';
+
+const stories = storiesOf(`${storybookPackageName(name)}/Catalog Components/Catalog Tile`, module);
+
+stories.addDecorator(
+  defaultTemplate({
+    title: 'Catalog Tile',
+    description: 'This is a tile that provides the layout desired for catalog items.'
+  })
+);
+
+stories.add(
+  'Catalog Tile',
+  withInfo({
+    source: true,
+    propTables: [CatalogTile, CatalogTileBadge]
+  })(() => (
+    <div style={{ display: 'flex' }}>
+      <CatalogTile
+        id="long-description-test"
+        featured
+        iconImg={pfBrand}
+        badges={[<CatalogTile.Badge type="fa" name="cog" title="Certified" id="certified" />]}
```

Comment:
Just want to verify this badge code is correct. It doesn't look like the other badge code, and it renders elements in the tile, but no visible icon.

Reply:
correct, left over from a previous attempt. Thank you!